sufia-models/app/models/concerns/sufia/works/curation_concern/with_generic_files.rb

```diff
@@ -5,7 +5,7 @@ module WithGenericFiles
 
       included do
         # This used to have a hasFile relation when in hydra-works.  That does not seem to exist so I am using hasPart instead
-        has_many :generic_files, predicate: ActiveFedora::RDF::Fcrepo::RelsExt.isPartOf, class_name: "GenericFile"
+        has_many :generic_files, predicate: ActiveFedora::RDF::Fcrepo::RelsExt.hasPart, class_name: "GenericFile"
```

Comment:
I see the comment on line 7 but i'm not clear on why Sufia should be doing this if Hydra::Works is.  Is it not?  Is this one of those areas where we have temporary code that will go away when we're more fully based on HW?  `hasPart` is clearly preferable to `isPartOf` here, but I would think HW already defines this relationship for us.